relay-server/src/utils/managed_envelope.rs

```diff
@@ -68,7 +68,7 @@ struct EnvelopeContext {
 }
 
 #[derive(Debug)]
-pub struct InvalidProcessingGroupType(pub ManagedEnvelope);
+pub struct InvalidProcessingGroupType(pub Box<ManagedEnvelope>);
```

Comment:
I assume this is because clippy complains about the type size, but this is just the same size as a `ManagedEnvelope`, so I don't think just wrapping this makes sense as the newtype doesn't add any size in the first place.

In this case I'd probably just ignore the lint, or we have to think about addressing `ManagedEnvelope`.

Reply:
Specifically, the complaint is that this particular variant causes `ProcessingError` to be at least as big as a `ManagedEnvelope`. Another option would be to put the box in the `InvalidProcessingGroup` variant of `ProcessingError` directly, but that would make the `From` impl slightly unergonomic.